src/pages/account/security.tsx

```diff
@@ -30,15 +31,18 @@ function Main() {
   const { t } = useTranslation();
   const { isProfileReady } = useUserProfile();
   return isProfileReady ? (
-    <div>
-      <Text component="h2" variant="h3" tw="mb-lg">
-        {t('account:security.login-title')}
-      </Text>
-      <ActionFieldGroup>
-        <EmailActionField />
-        <PasswordActionField />
-      </ActionFieldGroup>
-    </div>
+    <TabGroup buttonGap="lg" theme="secondary">
```

Comment:
Not intended to be used here. Each section should be navigated by URL. The reason is the aside can be different for every tab.

Reply:
> Not intended to be used here.

 On Airbnb website, these sections are all in a single page, I just make into a Tab structure so the user doesn't have to scroll.

> Each section should be navigated by URL.

This can be achieve by using the Tab in the controlled mode with the `selectedTab` and `onSelectedTabChange` props to control the currently selected tab.

Something like this:

```jsx
const [selectedTab, setSelectedTab] = useState(0);

return (
    <TabGroup buttonGap="lg" theme="secondary" selectedTab={selectedTab} onSelectedTabChange={setSelectedTab}>
      <Tab label={t('account:security.login-title')}>
        <ActionFieldGroup>
          <EmailActionField />
          <PasswordActionField />
        </ActionFieldGroup>
      </Tab>
      <Tab label={t('account:security.social-account-title')}>
        Tab 2 content
      </Tab>
      <Tab label={t('account:security.history-title')}>Tab 3 content</Tab>
    </TabGroup>
);
```

> The reason is the aside can be different for every tab.

The external state can control the content of the aside as well.

Reply:
If it's only for grouping then that's fine, but I don't recommend grouping content on 3 different pages into one component and controlling them with Tab. Tab should only be used to manage its own content, not the surrounding.

Reply:
> If it's only for grouping then that's fine, but I don't recommend grouping content on 3 different pages into one component and controlling them with Tab. 

They're actually one page, not multiple page. Here's what Airbnb's Login and Secruity page looks like:

![image](https://user-images.githubusercontent.com/54929186/144766223-7187e48f-fa5c-41cc-8de5-880596d82c87.png)

If I understand you correctly, you want to have the styling of the Tab structure but each tab button will actually navigate to one of the following URL?

* `/account/security/login`
* `/account/security/social-accounts`
* `/account/security/history`

We can do that as well... though I just don't unnerstand why this is a better option than the Tab given that in the original Airbnb, they have these all in 1 page.

Reply:
> If it's only for grouping then that's fine

No, I meant in this case it's fine.

Reply:
What do you mean by grouping?

Reply:
The same idea of your grouping. I meant this change can be resolved already.